apps/authentication-portal/src/main/webapp/login.jsp

```diff
@@ -18,6 +18,9 @@
 
 <%@ page language="java" contentType="text/html; charset=UTF-8" pageEncoding="UTF-8" %>
 <%@ page import="org.wso2.carbon.identity.application.authentication.endpoint.util.AuthContextAPIClient" %>
+<%@ page import="org.wso2.carbon.identity.application.authentication.framework.util.FrameworkUtils" %>
+<%@ page import="org.wso2.carbon.identity.application.authentication.framework.config.ConfigurationFacade" %>
+<%@ page import="org.wso2.carbon.identity.application.authentication.framework.config.model.ExternalIdPConfig" %>
```

Comment:
Any usages of these imports in the file?

Reply:
Fixed 9b4b2bc25c47eef914238e8c4592fd165da371c6